x-pack/plugins/timelines/public/components/t_grid/body/index.tsx

```diff
@@ -230,6 +231,12 @@ export const BodyComponent = React.memo<StatefulBodyProps>(
             $rowCount={data.length}
             $totalPages={totalPages}
           >
+            <TimelineToolbar
```

Comment:
Thanks for adding this here to make it easy to desk test the PR! To avoid any confusion for folks running the o11y app from `master` over the next few days (before we're fully integrated with EuiDataGrid), would you be willing to remove `TimelineToolbar` from here before merging?